index.html

```diff
@@ -6,14 +6,231 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    
+    <header>
+             <div class="container">
+                <div class="row">
+                    <div class=" top col-12">
```

Comment:
Do you need to set this to 12 columns, since you're immediately splitting it into 3 and 7 afterwards? I would re-structure it to something like this: 
```
<div class="container">
    <div class="row top">
        <figure class="logo col-5">
            <img src="./images/logo1.png" alt="logo" />
        </figure>

        <nav class="top-navigation col-7">
            <ul>
                //...
            </ul>
        </nav>
    </div>
</div>
```
(note that I have changed the number of columns (5 and 7) so that they add up to 12. The number of columns inside a `row` should always add up to 12)